### PR TITLE
BatchAI. Multiple documentation related improvements. No functional changes.

### PR DESCRIPTION
**Pull request by AlexanderYukhanov**
Multiple minor documentation related corrections. No functional changes.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 1]
```diff
@@ -1259,8 +1259,8 @@
           "type": "string",
           "readOnly": true,
           "x-nullable": false,
-          "title": "Specifies the provisioning state of the file server.",
-          "description": "Possible values: creating - The fileServer is just getting created. Updating - The file server creation has been accepted and it is getting updated. Deleting - The user has requested that the cluster be deleted, but the delete operation has not yet completed. Failed - The file server creation has failed with the specified errorCode. Details about the error code are specified in the message field. Succeeded - The file server creation has succeeded.",
+          "title": "Specifies the provisioning state of the File Server.",
+          "description": "Possible values: creating - The File Server is getting created. Updating - The File Server creation has been accepted and it is getting updated. Deleting - The user has requested that the File Server be deleted, and it is in the process of being deleted. Failed - The File Server creation has failed with the specified errorCode. Details about the error code are specified in the message field. Succeeded - The File Server creation has succeeded.",
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Does "creating" need to be capitalized (in the "Possible values: creating" part)?  The other values are capitalized.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 2]
```diff
@@ -1375,7 +1375,7 @@
           "title": "The scale for the cluster by autoscale settings"
         }
       },
-      "description": "Specifies the scale settings for the cluster. At least one of manual and autoScale settings must be specified. Only one of manual and autoScale settings can be specified."
+      "description": "At least one of manual or autoScale settings must be specified. Only one of manual or autoScale settings can be specified. if autoScale settings are specified, the system automatically scales the cluster up and down (within the supplied limits) based on the pending jobs on the cluster."
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: The word "if" at the start of the third sentence should be capitalized.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 3]
```diff
@@ -1621,14 +1621,14 @@
         },
         "scaleSettings": {
           "$ref": "#/definitions/ScaleSettings",
-          "title": "Desired scale for the cluster"
+          "title": "Desired scale for the Cluster"
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Missing period at end of sentence.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 4]
```diff
@@ -1950,7 +1950,7 @@
             "$ref": "#/definitions/EnvironmentSetting"
           },
           "title": "Additional environment variables to be passed to the job.",
-          "description": "Batch AI services sends the following environment variables for all jobs: AZ_BATCHAI_INPUT_id, AZ_BATCHAI_OUTPUT_id, AZ_BATCHAI_NUM_GPUS_PER_NODE, For distributed TensorFlow jobs, following additional environment variables are set by the Batch AI Service: AZ_BATCHAI_PS_HOSTS, AZ_BATCHAI_WORKER_HOSTS"
+          "description": "Batch AI services sets the following environment variables for all jobs: AZ_BATCHAI_INPUT_id, AZ_BATCHAI_OUTPUT_id, AZ_BATCHAI_NUM_GPUS_PER_NODE, For distributed TensorFlow jobs, following additional environment variables are set by the Batch AI Service: AZ_BATCHAI_PS_HOSTS, AZ_BATCHAI_WORKER_HOSTS"
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: I see that the phrase "Batch AI" was removed in a lot of other places but not here, is that intentional?

Comment (jhendrixMSFT) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Also no period at end of sentence.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 5]
```diff
@@ -2929,7 +2929,7 @@
       "required": false,
       "type": "string",
       "x-ms-parameter-location": "method",
-      "description": "list of files.",
+      "description": "An OData $select clause. Used to select the properties to be returned in the GET respnose",
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Missing period at end of sentence.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 6]
```diff
@@ -2940,7 +2940,7 @@
       "required": false,
       "type": "string",
       "x-ms-parameter-location": "method",
-      "description": "An OData $filter clause.",
+      "description": "An OData $filter clause.. Used to filter results that are returned in the GET respnose",
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Missing period at end of sentence.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 7]
```diff
@@ -2967,7 +2967,7 @@
       "required": true,
       "type": "string",
       "x-ms-parameter-location": "method",
-      "description": "Id of the job output directory.",
+      "description": "Id of the job output directory. This is the OutputDirectory-->id parameter that is given by the user during Create Job",
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Missing period at end of sentence.

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json [hunk 8]
```diff
@@ -1182,7 +1182,7 @@
           "items": {
             "type": "string"
           },
-          "title": "List of source IPs range which would be allowed to perform SSH with the VMs.",
+          "title": "List of source IPs range to be allowed for SSH connection to VMsfs.",
```
Comment (jhendrixMSFT) [inline specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Is "VMsfs" a type-o?

Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: thanks! done

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 2 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: done

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 3 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: thanks, done

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 5 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: thanks, done

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 4 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Batch AI was removed before Cluster, Job and File Server. Like it was "Batch AI Cluster", now it's "Cluster".  But Batch AI service remains.
added the period

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 1 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: thanks, that's the opposite. fixed

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 6 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: thanks, done

File: specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json (hunk 7 above)
Comment (AlexanderYukhanov) [reply specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json]: Done, thanks

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**